src/dls_bluesky_core/stubs/flyables.py

```diff
@@ -0,0 +1,28 @@
+import bluesky.plan_stubs as bps
+from bluesky.protocols import Flyable
+
+from dls_bluesky_core.core import MsgGenerator, group_uuid
+
+
+def fly_and_collect(
```

Comment:
I think this is complicated enough to need a docstring

Reply:
done